src/ports/postgres/modules/svm/svm.py_in

```diff
@@ -440,55 +595,68 @@ def _process_epsilon(is_svc, args):
 def _extract_params(schema_madlib, params, module='SVM'):
     # NOTICE: the type of values in params_default should be consistent with
     # the types specified in params_types
-    params_default = {'init_stepsize': 0.01,
-                      'decay_factor': 0.9,
-                      'max_iter': 100,
-                      'tolerance': 1e-10,
-                      'lambda': 1.0,
-                      'norm': 'L2',
-                      'n_folds': 0,
-                      'epsilon': 0.01,
-                      'eps_table': ''}
-
-    params_types = {'init_stepsize': float,
-                    'decay_factor': float,
-                    'max_iter': int,
-                    'tolerance': float,
-                    'lambda': list,
-                    'norm': str,
-                    'n_folds': int,
-                    'epsilon': float,
-                    'eps_table': str}
+    params_default = {
+                        'init_stepsize': [0.01],
+                        'decay_factor': [0.9],
+                        'max_iter': [100],
+                        'tolerance': 1e-10,
+                        'lambda': [0.01],
+                        'norm': 'L2',
+                        'n_folds': 0,
+                        'validation_result': '',
+                        'epsilon': [0.01],
+                        'eps_table': ''}
+
+    params_types = {
+                        'init_stepsize': list,
+                        'decay_factor': list,
+                        'max_iter': list,
+                        'tolerance': float,
+                        'lambda': list,
+                        'norm': str,
+                        'n_folds': int,
+                        'validation_result': str,
+                        'epsilon': list,
+                        'eps_table': str}
 
     params_vals = extract_keyvalue_params(params,
                                           params_types,
                                           params_default)
-
     if params_vals['n_folds'] < 0:
```

Comment:
let's use the `_assert` format to maintain consistency. I would like to add another PR to change all `_assert` to `assert_`